.ci/windows/test.bat

```diff
@@ -1,8 +1,24 @@
+::
+:: This script runs the dotnet build without the Full Framework projects
+::
+dotnet sln remove sample/AspNetFullFrameworkSampleApp/AspNetFullFrameworkSampleApp.csproj
+dotnet sln remove src/Elastic.Apm.AspNetFullFramework/Elastic.Apm.AspNetFullFramework.csproj
+dotnet sln remove test/Elastic.Apm.AspNetFullFramework.Tests/Elastic.Apm.AspNetFullFramework.Tests.csproj
+dotnet sln remove test/Elastic.Apm.SqlClient.Tests/Elastic.Apm.SqlClient.Tests.csproj
+dotnet sln remove test/Elastic.Apm.EntityFramework6.Tests/Elastic.Apm.EntityFramework6.Tests.csproj
+dotnet sln remove test/Elastic.Apm.MongoDb.Tests/Elastic.Apm.MongoDb.Tests.csproj
+
+:: Remove startup hooks tests, which are tested separately- require agent zip to be built
+dotnet sln remove test/Elastic.Apm.StartupHook.Tests/Elastic.Apm.StartupHook.Tests.csproj
+
+:: Remove profiler tests, which are tested separately- require profiler to be built
+dotnet sln remove test/Elastic.Apm.Profiler.Managed.Tests/Elastic.Apm.Profiler.Managed.Tests.csproj
+
 ::
 :: This script runs the tests and stored them in an xml file defined in the
 :: LogFilePath property
 ::
-dotnet test -c Release --no-build ^
+dotnet test -c Release ^
```

Comment:
Out of curiosity, if the test.bat does also run the build, are the `dotnet sln remove` needed for the building?

Reply:
Yes, that's intentional.

Those projects that we remove with `dotnet sln remove` need more than what we have in this context (e.g. ASP.NET Classic), so the `dotnet` tool isn't enough for those. We use those projects in the `Windows .NET Framework` stage.

I only know this from experience :) If we don't remove those, it'll fail. Prior to this PR removing happened in the `dotnet.bat` file.

Reply:
Thanks Greg for the clarification :)